### PR TITLE
Varnish cookie expiry fixes

### PR DESCRIPTION
**Pull request by torarvid**

### CODE REVIEW

File: src/identity.js [hunk 1]
```diff
@@ -267,10 +267,14 @@ export class Identity extends EventEmitter {
     /**
      * Set the Varnish cookie (`SP_ID`) when hasSession() is called. Note that most browsers require
      * that you are on a "real domain" for this to work — so, **not** `localhost`
+     * @param {number} [expiresIn] Override this to set number of seconds before the varnish cookie
+     * expires. The default is to use the same time that hasSession responses are cached for
      * @returns {void}
      */
-    enableVarnishCookie() {
+    enableVarnishCookie(expiresIn = 0) {
+        assert(Number.isInteger(expiresIn), `'expiresIn' must be an integer`);
```
Comment (marjuszkiewicz) [inline src/identity.js]: `expiresIn` should be positive integer, so maybe check if it is `>= 0`

Comment (torarvid) [reply src/identity.js]: The idea here was that I use the "falsy-ness" in JS as a test below, where I do `const expires = this.varnishExpiresIn || sessionData.expiresIn`. So I use `0` as a default value, basically meaning the same thing as `undefined` in practice. Also, it makes it easy to keep a compatible signature, with having `0` as a default value for the function (so people can still use it as `enableVarnishCookie()`). Sound ok?

Comment (marjuszkiewicz) [reply src/identity.js]: yes I saw it, just wonder what will happen if somebody will call `enableVarnishCookie(-1)`, then expires will be set to `-1` but maybe we shouldn't care about this

Comment (torarvid) [reply src/identity.js]: Ok. I'll add a check to be sure. Thanks 👍

**Pull request merged**